aea/cli/test.py

```diff
@@ -437,6 +437,13 @@ def load_package(
         )
         configuration.directory = package_dir
         load_aea_packages_recursively(configuration, package_path_finder, root_packages)
+    else:
+        perform_load_aea_package(
+            dir_=package_dir,
+            author=package_dir.parent.parent.name,
+            package_type_plural=PackageType(package_type).to_plural(),
+            package_name=package_dir.name,
+        )
```

Comment:
This addresses https://github.com/valory-xyz/open-autonomy/actions/runs/3819359169/jobs/6496927078#step:7:9939 failure. We load agent packages in same manner as we load the component packages before running the tests

Reply:
Where is the test for this branch? Why was it not required before?

Reply:
We did not load the agent package before because there was no code part of the agent component similar to other components so we could run the test without loading the agent package. 

When I was recreating this issue locally I found out that this issue only occurs in one specific condition that is if you run the tests in a tox env and all packages available. When I ran tests for agents only by modifying the test command they ran without any issue and when I ran the same command we run in the tox env normally they ran without any issues as well.  

So my assumption is since we're loading the components other than the agents the references in the `sys.modules` does not contain references to the `packages.author.components` and not the `packages.author.agents` which is the reason why the pytest was throwing import error. 

Now this did not happen before, this started happening after the release of `1.27.0` and we haven't changed anything in the test command except for the https://github.com/valory-xyz/open-aea/pull/504 which does not look like could've caused the issue so this is still a mystery for me why this started happening suddenly